ilastik/applets/cropping/cropSelectionGui.py

```diff
@@ -295,12 +295,12 @@ def newCrop(self):
         selectedRow = ncrops-1
         color1 = self._cropControlUi.cropListModel[selectedRow].brushColor()
         color2 = self._cropControlUi.cropListModel[selectedRow].pmapColor()
-        self.topLevelOperatorView.Crops.value[unicode(self._cropControlUi.cropListModel[selectedRow].name)] = {
-            unicode("time"): (self.topLevelOperatorView.MinValueT.value, self.topLevelOperatorView.MaxValueT.value),
-            unicode("starts"): self.editor.cropModel.get_roi_3d()[0],
-            unicode("stops"): self.editor.cropModel.get_roi_3d()[1],
-            unicode("cropColor"): (color1.red(), color1.green(),color1.blue()),
-            unicode("pmapColor"): (color2.red(), color2.green(),color2.blue())
+        self.topLevelOperatorView.Crops.value[str(self._cropControlUi.cropListModel[selectedRow].name)] = {
+            str("time"): (self.topLevelOperatorView.MinValueT.value, self.topLevelOperatorView.MaxValueT.value),
+            str("starts"): self.editor.cropModel.get_roi_3d()[0],
+            str("stops"): self.editor.cropModel.get_roi_3d()[1],
+            str("cropColor"): (color1.red(), color1.green(),color1.blue()),
+            str("pmapColor"): (color2.red(), color2.green(),color2.blue())
```

Comment:
Sounds nice. If you are thinking of generalizing this to complete code, I just read a word of caution here - https://pythonhosted.org/six/#six.u
"Note On Python 2, u() doesn’t know what the encoding of the literal is. Each byte is converted directly to the unicode codepoint of the same value. Because of this, _**it’s only safe to use u() with strings of ASCII data**_"